doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,599 @@
+automatically at the time of RTVM creation. Refer to :ref:`rdt_configuration`
+for details on RDT configuration and :ref:`hv_rdt` for details on RDT
+high-level design.
+
+Set up the core allocation for the RTVM
+---------------------------------------
+
+In our recommended configuration, two cores are allocated to the RTVM:
+core 0 for housekeeping and core 1 for RT tasks. In order to achieve
+this, follow the below steps to allocate all housekeeping tasks to core 0:
+
+#. Launch the RTVM::
+
+   # /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+
+#. Log in to the RTVM as root and run the script as below:
+
+   .. code-block:: bash
+
+      #!/bin/bash
+      # Copyright (C) 2019 Intel Corporation.
+      # SPDX-License-Identifier: BSD-3-Clause
+      # Move all IRQs to core 0.
+      for i in `cat /proc/interrupts | grep '^ *[0-9]*[0-9]:' | awk {'print $1'} | sed 's/:$//' `;
+      do
+          echo setting $i to affine for core zero
+          echo 1 > /proc/irq/$i/smp_affinity
+      done
+
+      # Move all rcu tasks to core 0.
+      for i in `pgrep rcu`; do taskset -pc 0 $i; done
+
+      # Change realtime attribute of all rcu tasks to SCHED_OTHER and priority 0
+      for i in `pgrep rcu`; do chrt -v -o -p 0 $i; done
+
+      # Change realtime attribute of all tasks on core 1 to SCHED_OTHER and priority 0
+      for i in `pgrep /1`; do chrt -v -o -p 0 $i; done
+
+      # Change realtime attribute of all tasks to SCHED_OTHER and priority 0
+      for i in `ps -A -o pid`; do chrt -v -o -p 0 $i; done
+
+      echo disabling timer migration
+      echo 0 > /proc/sys/kernel/timer_migration
+
+   .. note:: You can ignore the error messages during the script running.
+
+Run cyclictest
+--------------
+
+#. Refer to the :ref:`troubleshooting section <enabling the network on RTVM>`   below that discusses how to enable the network connection for RTVM.
+
+#. Launch the RTVM and log in as root.
+
+#. Install the ``cyclictest`` tool::
+
+   # swupd bundle-add dev-utils --skip-diskspace-check
+
+#. Use the following command to start cyclictest::
+
+   # cyclictest -a 1 -p 80 -m -N -D 1h -q -H 30000 --histfile=test.log
+
+   Parameter descriptions:
+
+    :-a 1:                           to bind the RT task to core 1
+    :-p 80:                          to set the priority of the highest prio thread
+    :-m:                             lock current and future memory allocations
+    :-N:                             print results in ns instead of us (default us)
+    :-D 1h:                          to run for 1 hour, you can change it to other values
+    :-q:                             quiet mode; print a summary only on exit
+    :-H 30000 --histfile=test.log:   dump the latency histogram to a local file
+
+Launch the Windows VM
+*********************
+
+#. Follow this :ref:`guide <using_windows_as_uos>` to prepare the Windows
+   image file, update the Service VM kernel, and then reboot with a new ``acrngt.conf``.
```

Comment:
Do we need to change the Service VM kernel we just installed above?

Reply:
@fuzhongl Please respond.

Reply:
Please remove 
update the Service VM kernel,

Reply:
This now says:

#. Follow this :ref:`guide <using_windows_as_uos>` to prepare the Windows
   image file and then reboot with a new ``acrngt.conf``.